ethereum-common/Cargo.toml

```diff
@@ -19,16 +19,18 @@ harness = false
 ethabi = "17"
 hex-literal = "0.3"
 num-bigint = "0.4"
-prost = "0.11"
-prost-types = "0.11"
+prost = "0.13"
+prost-types = "0.13"
+prost-reflect = "0.14.4"
 # Use latest from https://crates.io/crates/substreams
-substreams = "0.5.14"
+substreams = "0.6.0"
 # Use latest from https://crates.io/crates/substreams-ethereum
-substreams-ethereum = "0.9"
+substreams-ethereum = "0.10"
 serde_qs = "0.12.0"
 serde = { version = "1.0", features = ["derive"] }
 anyhow = "1.0"
 regex = "1.10.4"
+base64 = "0.22.1"
```

Comment:
Add this to `dev-dependencies` section instead,  see https://stackoverflow.com/a/29857478/697930